secretariat/admin.py

```diff
@@ -14,6 +14,24 @@ class MembershipInlineForOrganisation(MembershipInline):
     verbose_name_plural = "Membres"
 
 
+class SynchronizedWithOutlineFilter(admin.SimpleListFilter):
+    title = "Synchronisé avec Outline"
+    parameter_name = "outline_sync"
+    field_name = "outline_uuid"
+
+    def lookups(self, request, model_admin):
+        return (
+            ("oui", "Oui"),
+            ("non", "Non"),
+        )
+
+    def queryset(self, request, queryset):
+        if self.value() == "oui":
+            return queryset.filter(**{f"{self.field_name}__isnull": False})
```

Comment:
Vous connaissez la syntaxe `**` ? Ça permet de "déballer" les objets pour les passer en paramètre à une fonction.
Donc par exemple `fonction(**{"x":12,"y":8})` est équivalent à `fonction(x=12,y=8)`.

Ici c'est pour appeler `filter` avec un paramètre dont je n'ai le nom que dans une variable.

Reply:
Oh mais ... en te voyant aire comme ça, je me demande si c'est pas la solution aussi à mon souci de switch/case d'hier soir ! Je testerai bientôt pour pouvoir switche/caser peut-être pendant l'avent of code

Reply:
L'advent of code c'est demaiiiin 🎉